<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Friends</class>
 <widget class="QWidget" name="Friends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="fish_icon">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fish-logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fish Buddies</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>321</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="postLayout"/>
  </widget>
  <widget class="QPushButton" name="homePreference_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Preference Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="add_a_friend">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>509</width>
      <height>59</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="title_input">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextUser_button">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Friend</string>
     </property>
    </widget>
    <widget class="QLabel" name="logWarning_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color : red
</string>
     </property>
     <property name="text">
      <string>Sadly this fish enthusiast is not apart of Fish Buddies yet!</string>
     </property>
    </widget>
    <widget class="QLabel" name="friends_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(90, 196, 61)</string>
     </property>
     <property name="text">
      <string>Hooray You're already Fish Buddies</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>461</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="load_list">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>See Other Fish Enthusiast</string>
   </property>
  </widget>
  <widget class="QLabel" name="Friends_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>My Fish Buddies</string>
   </property>
  </widget>
  <widget class="QListWidget" name="button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="217">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="63">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <item>
    <property name="text">
     <string>
                  Home
</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>66</red>
       <green>118</green>
       <blue>253</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="DiagCrossPattern">
      <color alpha="255">
       <red>250</red>
       <green>250</green>
       <blue>250</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
